openbb_terminal/core/plots/backend.py

```diff
@@ -387,6 +387,75 @@ def close(self, reset: bool = False):
 
         super().close()
 
+    async def get_results(self, description: str) -> dict:
+        """Wait for completion of interactive task and return the data.
+
+        Parameters
+        ----------
+        description : str
+            Description of the task to console print while waiting.
+
+        Returns
+        -------
+        dict
+            The data returned from pywry backend.
+        """
+        console.print(
+            f"[green]{description}[/]\n\n"
+            "[yellow]If the window is closed you can continue by pressing Ctrl+C.[/]"
+        )
+        while True:
+            try:
+                data: dict = self.recv.get(block=False) or {}
+                if data.get("result", False):
+                    return json.loads(data["result"])
+            except Exception:  # pylint: disable=W0703
+                pass
+
+            await asyncio.sleep(1)
+
+    def call_hub(self, login: bool = True) -> Optional[dict]:
+        """Call the hub to login or logout.
+
+        Parameters
+        ----------
+        login : bool, optional
+            Whether to login or logout, by default True
+
+        Returns
+        -------
+        Optional[dict]
+            The user data if login was successful, None otherwise.
+        """
+        self.loop.run_until_complete(self.check_backend())
+        endpoint = {True: "login", False: "logout"}[login]
+
+        outgoing = dict(
+            json_data=dict(url=f"https://my.openbb.dev/{endpoint}?pywry=true"),
```

Comment:
needs to be changed to .co for production

Reply:
yes but first the frontend changes need to be pushed to `"MY".openbb.co`